Motion Amendments 2019 - Committe/Team Senior Member (#26)
thewca/wca-documents: documents/motions/02.2018.1 - Definitions.md

    </tr>
    <tr>
      <td>Committee</td>

>       <td>A Committee of the WCA that has been appointed by the WCA under the terms of the WCA Bylaws or any of the Motions. Leaders and Senior Members of Committees are WCA Staff and rest of Committee members are WCA External Staff.</td>

Reviewer: The definition of "Committee" uses the word "Committee"?

Reviewer 2: I think this is okay. It's not so much the definition of committee that is important, as much as to make it clear that when we use the word committee, this is what we are referring to.

      <td>Senior Delegate</td>
      <td>A WCA Staff Member and Full Delegate who has been appointed by the WCA to manage a group of WCA Delegates.</td>
    </tr>
    <tr>
      <td>Senior Member</td>

>       <td>A member of one or more Committee(s) or Team(s) who has been a member in at least one of the Committee(s) Team(s) for a year and has been appointed by its Leader.</td>

Reviewer 3: ```suggestion
      <td>A Committee or Team Member who has been on it for at least a year and has been appointed by its Leader.</td>
```

I think this looks clearer. Also, in my opinion, a Senior member should be a per-Committee/Team role, so I suggested accordingly.

          <li>Committee/Team Leader</li>
          <li>Senior Delegate</li>
          <li>Full Delegate</li>

>           <li>Senior Member of a Committee/Team</li>

Reviewer 3: The order here seems weird to me.

    </tr>
    <tr>
      <td>Team</td>

>       <td>A team of the WCA that has been appointed by the WCA under the terms of these Bylaws or any of the Motions. Leaders and Senior Members of teams are WCA Staff and rest of team members are WCA External Staff.</td>

Reviewer: nit: In the definition of "Committee" earlier, you capitalized the `c`, do  you want to capitalize the the two `t`s here?

Also seems weird to be using the word "team" to define the term "team" here.

>       <td>A team of the WCA that has been appointed by the WCA under the terms of these Bylaws or any of the Motions. Leaders and Senior Members of teams are WCA Staff and rest of team members are WCA External Staff.</td>

Reviewer 3: ```suggestion
      <td>A Team of the WCA that has been appointed by the WCA under the terms of these Bylaws or any of the Motions. Leaders and Senior Members of Teams are WCA Staff and rest of Team Members are WCA External Staff.</td>
```
